<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌面文件管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/image/R-C.png</normaloff>:/image/R-C.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QPushButton" name="btnFresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnDelete">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnUpload">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
